src/platform/qpg6100/BLEManagerImpl.cpp

```diff
@@ -435,6 +553,26 @@ void BLEManagerImpl::HandleTXCharCCCDWrite(qvCHIP_Ble_AttsCccEvt_t * pEvt)
     // Determine if the client is enabling or disabling indications.
     indicationsEnabled = (pEvt->value != 0);
 
+    // If the client has requested to enabled indications
+    if (indicationsEnabled)
+    {
+        // If indications are not already enabled for the connection...
+        if (!IsSubscribed(pEvt->hdr.param))
+        {
+            // Record that indications have been enabled for this connection.  If this fails because
```

Comment:
Please finish the 2nd sentence or remove it.

Reply:
Removed the last part.